src/main/scala/ayai/factories/GraphFactory.scala

```diff
@@ -3,44 +3,34 @@ package ayai.factories
 import ayai.gamestate.RoomWorld
 import ayai.components.{Position, Bounds}
 
-// Merges tilemap and world into a traverseable graph for AI purposes
+// Merges tilemap and world into a traversable graph for AI purposes
 
 object NodeState extends Enumeration {
   type NodeState = Value
   val IMPASS, SLOW, NORMAL = Value
 }
 
-import NodeState.{NodeState, IMPASS, NORMAL}
+import NodeState.{NodeState, IMPASS, NORMAL, SLOW}
 
-object Node {
-  def apply(state: NodeState) = new Node(state)
+case class Node(val state: NodeState) {
+  def printableState = state match {
```

Comment:
can we just override `toString` with the contents of this method instead of something completely new?